identity/README.md

```diff
@@ -48,3 +55,14 @@ devOverrides {
   id.apiClientToken=frontend-dev-client-token
 }
 ```
+
+You can configure [identity-frontend in the same way].
+
+[identity-frontend in the same way]: https://github.com/guardian/identity-frontend#running-against-dev-identity-api
+
+### Proxying local Identity API requests to CODE
+
+You can also use `idapi-code-proxy.thegulocal.com` locally to proxy requests to
+CODE. This is necessary if you want requests to CODE identity API to work from
```

Comment:
> This is necessary if you want requests to CODE identity API to work

Unless you're signed in on CODE in the same browser?

Reply:
Yes good point, that's true. To me this feels like the more more "correct" approach, since things can't get out of sync in the same way and will behave more predictably (i.e. signed in as different users locally vs on CODE, signing out locally will clear the expected cookies).